app/code/Bulbulatory/Recommendations/Helper/Recommendation.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+
+namespace Bulbulatory\Recommendations\Helper;
+
+use Bulbulatory\Recommendations\Model\RecommendationRepository;
+use Bulbulatory\Recommendations\Model\Source\Status;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Customer\Model\Customer;
+use Magento\Framework\Phrase;
+
+/**
+ * Class Recommendation
+ * Helper class for business logic related methods for recommendations module
+ * @package Bulbulatory\Recommendations\Helper
+ */
+class Recommendation extends AbstractHelper
+{
+    /**
+     * @var RecommendationRepository
+     */
+    private $recommendationRepository;
+
+    /**
+     * Recommendation constructor.
+     * @param Context $context
+     * @param RecommendationRepository $recommendationRepository
+     */
+    public function __construct(Context $context, RecommendationRepository $recommendationRepository)
+    {
+        parent::__construct($context);
+        $this->recommendationRepository = $recommendationRepository;
+    }
+
+    /**
+     * Returns integer value of customer discount by number of his confirmed recommendations
+     * @param int $confirmedRecommendations
+     * @return int
+     */
+    public function getDiscountValueForCustomer(int $confirmedRecommendations)
+    {
+        if ($confirmedRecommendations < 10) {
+            return 0;
+        } else if ($confirmedRecommendations < 20) {
+            return 10;
+        } else {
+            return 20;
+        }
+    }
```

Comment:
Za każde 10 poleceń powinno być 5% zniżki, a nie 10. No i 20 poleceń to nie jest maksymalna liczba, jaka może być, co jeśli użytkownik ma 50 zatwierdzonych poleceń? Wtedy zniżka powinna wynosić 25%.